examples/plugins/project.json

```
@@ -53,6 +53,14 @@
         "^build",
         { "projects": ["cli"], "target": "build" }
       ]
+    },
+    "run-help": {
+      "command": "npx dist/packages/cli help --config=examples/plugins/code-pushup.config.ts",
```

Why is there a `--config` parameter added to `help` command?
```suggestion
      "command": "npx dist/packages/cli help",
```